internal/entities/toolsconfig/tools_config.go

```diff
@@ -81,17 +86,38 @@ func (t *ToolsConfigsStruct) ToMap() MapToolConfig {
 	}
 }
 
-func ParseInterfaceToMapToolsConfig(input interface{}) (output MapToolConfig) {
-	outputStruct := ToolsConfigsStruct{}
-	bytes, err := json.Marshal(input)
+// Default return the default configuration of tools.
+//
+// The default configuration is enabled for all tools.
+func Default() ToolsConfig {
+	return (&toolsConfig{}).toMap()
+}
+
+// MustParseToolsConfig parse a input to ToolsConfig.
+//
+// If some error occur the default values will be returned and the error
+// will be logged.
+func MustParseToolsConfig(input map[string]interface{}) (output ToolsConfig) {
+	cfg, err := parseToolsConfig(input)
 	if err != nil {
 		logger.LogErrorWithLevel(messages.MsgErrorParseStringToToolsConfig, err)
-		return outputStruct.ToMap()
+		return Default()
 	}
-	err = json.Unmarshal(bytes, &outputStruct)
+	return cfg
```

Comment:
the function names the output value `func MustParseToolsConfig(input map[string]interface{}) (output ToolsConfig)` but returns an cfg var, i think it's confusing. Please use output as cfg or change function sig to `func MustParseToolsConfig(input map[string]interface{}) ToolsConfig`

Reply:
I let this pass. Thanks!